.travis.yml

```diff
@@ -4,7 +4,7 @@ services: docker
 
 before_install:
  - CONTAINER=$(docker run -it -d -v $PWD:/build ubuntu:16.04 /bin/bash)
- - docker exec -i $CONTAINER /bin/bash -c "apt-get update -qq && apt-get install -y -qq gnupg git equivs lintian devscripts build-essential bc python-future dh-systemd dh-autoreconf wget libglfw3-dev cmake pkg-config libusb-1.0-0-dev"
+ - docker exec -i $CONTAINER /bin/bash -c "apt-get update -qq && apt-get install -y -qq gnupg git equivs lintian devscripts build-essential bc python-future dh-systemd dh-autoreconf wget libglfw3-dev cmake pkg-config libusb-1.0-0-dev libavahi-client-dev libavahi-core-dev libavahi-glib-dev gstreamer-1.0 libgstreamer-plugins-base1.0-dev libgstrtspserver-1.0-dev"
```

Comment:
The list of packages added to this list differs from the one at https://github.com/intel-aero/packages/pull/16/commits/cbe5d860b32a718c7a573a7ba6fadb82a2d6061d#diff-f1ac8795020ceffc2a0af67bf2b3f5d4 (I am obviously considering only the CSD-specific packages)

Reply:
This list is as per the dependency list in:  https://github.com/01org/camera-streaming-daemon/blob/master/configure.ac